### DISCUSSION TITLE
PatchTool 6.38 5/13 更新翻譯至6.3 修正bug

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#325] [Announcements]
更新漢化至中國服6.3、修正翻譯和簡繁轉換。適用6.38版本。

修正上版本的csv格式問題。

檔案：

- FFXIVChnTextPatch-20230513-fixed.rar：本體。一般而言，下載這個即可。

選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20230513-fixed.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- [backup-20230405.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.2.1/backup-20230405.rar)：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/'>PatchTool 6.38 5/13 更新翻譯至6.3 修正bug</a>.</em>

**Comment by Logan8246EX** [comment]
感謝

**Comment by tonyjediex** [comment]
終於等到6.3了 謝謝翻譯的大哥~

**Comment by a61456711** [comment]
（我尚未使用 6.38最新版本、仍使用上一版，不確定有沒有修正，回報一下=/）

80等支線任務「呼喚等待之人」，沒有提示要在說話頻道輸入日文或英文，嘗試輸入英文「Coo」沒有反應，去掉漢化包再輸入才能正常進入下一段

**Comment by Bos-hunting** [comment]
今天國際服有6.4的更新，希望會有暫時的兼容版，感謝

**Reply by GpointChen** [reply]
已更新。

**Comment by memory80626** [comment]
請問安裝漢化到最後跳這個問題的話，是哪邊有問題呢QQ
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/134389443/a0c1040c-fddb-435e-b154-be60a765a46f)

**Reply by a61456711** [reply]
6.4更新後，嘗試用現版本漢化就會跳出錯誤，我也是有跳出這訊息，等候一下吧……

**Reply by memory80626** [reply]
想說是不是有什麼步驟我沒做好導致出錯，更新前還特別先按還原後才更新
看來只能靜候作者修正了，謝謝告知

**Reply by ReyJuiHsu** [reply]
我在全新乾淨的 6.4 版本執行漢化程式一樣會跳錯誤
但可以正常進遊戲以及看到中文(英文>中文)
FYI~

**Reply by GpointChen** [reply]
已更新。

Stain.EXH在6.4更新後多了一欄（column）文字但在舊版CSV中抓不到資料所致。
